BambooTracker.desktop

```
@@ -2,9 +2,11 @@
 Name=BambooTracker
 GenericName=OPNA Tracker
 GenericName[jp]=OPNAのトラッカーです
-Comment=A tracker for the YM2608 (OPNA), which was used in NEC PC-8801/9801 series of computers.
+GenericName=Tracker OPNA
```

I think you forgot the locale specification
```suggestion
 GenericName[fr]=Tracker OPNA
```